packages/langium/src/grammar/langium-grammar.langium

```diff
@@ -204,10 +212,11 @@ CharacterRange infers AbstractElement:
     {infer CharacterRange} left=Keyword ('..' right=Keyword)?;
 
 FeatureName returns string:
-    'current' | 'entry' | 'extends' | 'false' | 'fragment' | 'grammar' | 'hidden' | 'import' | 'interface' | 'returns' | 'terminal' | 'true' | 'type' | 'infer' | 'infers' | 'with' | PrimitiveType | ID;
+    'Infinity' | 'NaN' | 'undefined' | 'current' | 'entry' | 'extends' | 'false' | 'fragment' | 'grammar' | 'hidden' | 'import' | 'interface' | 'returns' | 'terminal' | 'true' | 'type' | 'infer' | 'infers' | 'with' | PrimitiveType | ID;
```

Comment:
This doesn't work: the keywords take higher precedence than the `NUMBER` token. And we have a validation forbidding the use of these names anyway because they are reserved names of the JavaScript runtime.